### PR TITLE
Login Page

### LINKED ISSUE #8: F1.2.1 The service shall allow users to log in using their registered email address and password.

### PR DESCRIPTION
**Pull request by wr1159**
Resolves #8 

## Changes
- Change landing page "get started" to login button
- Add login form 
  -  If admin user, redirect to user-management page, else redirect to landing page (TBC)

### CODE REVIEW

File: frontend/components/auth/login-page.tsx
Comment (SelwynAng) [inline frontend/components/auth/login-page.tsx:1]: Slight nitpick with the naming of this component file. Shall we name it `login-form.tsx` instead to reflect the name of the component itself. Also, the name of the function here is `LoginForm`

File: frontend/components/auth/login-page.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          </div>
+          <div className="grid gap-2">
+            <div className="flex items-center">
+              <Label htmlFor="password">Password</Label>
+              {/* TODO: Link to forget password */}
+              <Link href="#" className="ml-auto inline-block text-sm underline">
+                Forgot your password?
+              </Link>
+            </div>
+            <Input
```
Comment (SelwynAng) [inline frontend/components/auth/login-page.tsx:65]: This is more a UX issue (which we can always handle later), but we should give the user an option to toggle the visibility of their password.

**Review by SelwynAng — APPROVED** [review]
LGTM aside from some small nitpicks!

File: frontend/components/auth/login-page.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const auth = useAuth();
+  const router = useRouter();
+
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    const user = await auth?.login(email, password);
+    if (user?.isAdmin) {
+      router.push("/admin-user-management");
```
Comment (jq1836) [inline frontend/components/auth/login-page.tsx:29]: Why do we redirect them to "/admin-user-management" page? Is it for the sake of testing that page? Might be better to just redirect to "/" so we don't forget it in the future

**Review by jq1836 — APPROVED** [review]
LGTM apart from a minor nitpick

File: frontend/components/auth/login-page.tsx (hunk 2 above)
Comment (wr1159) [reply frontend/components/auth/login-page.tsx:29]: Thought would be better UX, like how when you login on other websites, you go to the actual content page

File: frontend/components/auth/login-page.tsx (hunk 1 above)
Comment (wr1159) [reply frontend/components/auth/login-page.tsx]: Handled now

File: frontend/components/auth/login-page.tsx
Comment (wr1159) [reply frontend/components/auth/login-page.tsx:1]: fixed

**Pull request merged**